portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,20 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ryan's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Ryan's Portfolio</h1>
+      <p>Hello! My name is Ryan Simone and I am currently a Google Step Intern. I am a rising junior at the University of California Santa Barbara pursuing a Bachelor's degree in Computer Science.</p>
```

Comment:
I'm not sure if these files have a line limit, but you should aim to keep it under 100 characters.  I would wrap the lines like this:

<p>
  This is the first line up to 100 characters
  continuing line etc etc up to 100
  continuing line etc etc up to 100.
</p>